proc_gradient.py

```diff
@@ -1,5 +1,4 @@
 from pyspecdata import *
-from scipy.optimize import leastsq,minimize,basinhopping
 fl = figlist_var()
 date = '200106'
 label_str = 'none'
```

Comment:
have never used and don't know purpose of... assuming alec uses for his diffusion and gradient studies

Reply:
yes .. if you're not checking that it works, might be worth removing it for now

Reply:
removing from proc_scripts?

Reply:
yes

Reply:
hmm, or just leave it for now and have it raise an exception at the beginning saying it's not tested and doesn't use the 1st and 2nd level functions -- then Alec could upgrade it once you document all your functions